recipe/build.sh

```diff
@@ -1,14 +1,20 @@
 #!/bin/bash
 
 # Get an updated config.sub and config.guess
-cp -r ${BUILD_PREFIX}/share/libtool/build-aux/config.* .
+cp $BUILD_PREFIX/share/gnuconfig/config.* .
 
 # Avoid overlinking. zlib and liblzma are private to libxml2
 find "${PREFIX}/lib" -name "*.la" -delete -print
 sed -i.bak -e 's/-llzma //g' -e 's/-lz //g' $PREFIX/bin/xml2-config
 
+# It looks like an missing symbol of libxml on osx-arm64
+if [[ ${target_platform} =~ .*arm64.* ]]; then
+    LDFLAGS="${LDFLAGS} -lxml2"
+fi
+
 ./configure --prefix=$PREFIX \
-            --with-libxml-prefix=$PREFIX
+            --with-libxml-prefix=$PREFIX \
+            --without-python
```

Comment:
I am not sure --without-python is actually used. I cannot find references of it the build files at least. But that shouldn't hurt.